maintainers/maintainer-list.nix

```diff
@@ -12602,5 +12602,15 @@
     github = "mbprtpmnr";
     githubId = 88109321;
   };
+  ametrine = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
Done.